pkg/nsx/services/inventory/build.go

```diff
@@ -89,7 +89,7 @@ func (s *InventoryService) BuildPod(pod *corev1.Pod) (retry bool) {
 		OriginProperties:        originProperties,
 		Status:                  status,
 	}
-
+	log.V(1).Info("Build pod", "current instance", containerApplicationInstance, "pre instance", preContainerApplicationInstance)
 	operation, _ := s.compareAndMergeUpdate(preContainerApplicationInstance, containerApplicationInstance)
```

Comment:
It looks like preContainerApplicationInstance is a value while containerApplicationInstance is a pointer?

Reply:
By the way, may I learn about why you use value for preContainerApplicationInstance? If there is record in the store, the pre will also not but nil even if it is a pointer?